tests/test_core.py

```diff
@@ -36,7 +36,8 @@
 
 
 def wasm_simd(f):
-  def decorated(self):
+  @wraps(f)
+  def decorated(self, *args):
```

Comment:
In other places we use `self, *args, **kwargs`

Reply:
Yes, that's even more correct. I'll do that.